eslint_temporary_suppressions.js

```diff
@@ -0,0 +1,1882 @@
+/*
+ * XXX: Temporary suppressions
+ *
+ * These rules are suppressed because we haven't yet fixed offending code.
+ *
+ * Want to help? Remove the suppression, fix any lint errors, and submit a PR.
+ */
+
+/** @type { import("eslint").Linter.Config[] } */
+export default [
+  /* Global rule suppressions */
+
+  {
+    rules: {
+      // Empty functions and blocks are useful (e.g `noop() {}`, `catch {}`) but can mask unintentionally omitted
+      // implementation. We should add explanatory comments like `// intentionally empty` and `// ignore error` in these
+      // scenarios to communicate intent.
+      'no-empty': 'off',
+      '@typescript-eslint/no-empty-function': 'off',
+
+      // We use these globals even though they're technically unstable features on our minimum
+      // supported node version.
+      'n/no-unsupported-features/node-builtins': [
+        'error',
+        {
+          ignores: ['FormData', 'ReadableStream', 'Response', 'Request', 'fetch', 'fs/promises.cp'],
+        },
+      ],
+    },
+  },
+
+  /* Per-file rule suppressions */
+
+  { ignores: ['functions-templates/**'] },
+  { ignores: ['site/**'] },
```

Comment:
🤔 are these really temporary?

Reply:
They are until we decide they're permanent!